ArraysAndLists/Program.cs

```diff
@@ -19,49 +16,99 @@ public static class ArraysAndLists
         // If more than two people like your post, it returns: "[Friend 1], [Friend 2] and [Number of Other People] others liked your post."
         public static string FacebookLikes(params string[] names)
         {
-            // Delete the below line when you are writing your code, don't forget to use your
-            // own return statement.
-            return default;
-        }
+            if (names.Length == 0)
+                return null;
+            if (names.Length == 1)
+                return names[0] + " liked your post.";
+            if (names.Length == 2)
+                return names[0] + " and " + names[1] + " liked your post.";
+            else
+                return names[0] + ", " + names[1] + " and " + (names.Length - 2) + " others liked your post.";
```

Comment:
Instead of using multiple return statements, a better option would be to create a variable that you change in each if statement, then return that singular statement at the end.